components/LinkWithOutPrefetch.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { AnchorHTMLAttributes, forwardRef } from "react";
+import { useRouter } from "next/router";
+
+const LinkWithOutPrefetch = forwardRef<
+  HTMLAnchorElement,
+  AnchorHTMLAttributes<HTMLAnchorElement>
+>(({ children, href, ...rest }, ref) => {
+  const { push } = useRouter();
+
+  const onClickPush = (e: React.MouseEvent<HTMLAnchorElement>) => {
+    e.preventDefault();
+    if (href) {
+      push(href);
+    }
+  };
+
+  return (
+    <a {...rest} onClick={onClickPush} href={href} ref={ref}>
```

Comment:
Btw is this OK? Does links in Next still work like they should? 
- aka it just updates the routes / doesn't do full page reload with these links

Shouldn't we rely on https://nextjs.org/docs/api-reference/next/link ?

Reply:
React Link Components not allow to turn completely off prefetch. even if we set prefetch={false}, it will does when on hover.

Reply:
Link is just abstraction for the router. the main reason why we use Link is that it's heavily modified and maintained a tag for SEO. 

so disable prefeching require to create a custom a tag component.

Reply:
I created new branch where I resolved the conflicts / and we have preview deployment. 
https://github.com/cohstats/coh3-stats/pull/161

I tried to do some research and indeed it looks like this is the only way to do this 👍